app/src/main/java/com/example/hackernews/callbacks/LoadCommentCallback.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.hackernews.callbacks
+
+import com.example.hackernews.models.Comment
+
+interface LoadCommentCallback {
```

Comment:
`LoadCommentCallback` and `LoadDataCallback` have the same structure, the only difference is data type so we can create a single generic callback to be used for similar cases. 

```kotlin
interface Callback<T> {
  fun onLoaded(data: T)
  fun onFailed(ex: Exception)
}

...

apiCall.loadComments(selectedNews.kids, object : Callback<Comment> {
   override fun onLoaded(comment: Comment) {}
   override fun onFailed(ex: Exception) {}
}
```